index.js

```diff
@@ -15,3 +15,128 @@ Array.from(closes).forEach((close) => {
     document.querySelector('#mobileNav').style.height = '0%';
   });
 });
+
+// popup window for works section
+const projects = [
+  {
+    project_id: 'project1',
+    name: 'Tonic',
+    experiance: ['CANOPY', 'Backend Dev', '2015'],
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription: 'Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop.png',
+  },
+  {
+    project_id: 'project2',
+    name: 'Multi-Post Stories',
+    experiance: ['FACEBOOK', 'Backend Dev', '2015'],
+    description:
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop-2.png',
+  },
+  {
+    project_id: 'project3',
+    name: 'Facebook 360',
+    experiance: ['Facebook', 'Backend Dev', '2015'],
+    description:
+      'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum? Dolor ipsam molestiae assumenda nisi natus voluptatum?',
+    image: 'images/Snapshoot-Portfolio-desktop3.jpg',
+  },
+  {
+    project_id: 'project4',
+    name: 'Uber Navigation',
+    experiance: ['Uber', 'Backend Dev', '2015'],
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    languages: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    popuplanguage: ['html', 'css', 'javaScript', 'github', 'ruby', 'bootstrap'],
+    popDescription:
+      'Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum Dolor ipsam molestiae assumenda nisi natus voluptatum',
+    image: 'images/Snapshoot-Porfolio-desktop4.png',
+  },
+];
```

Comment:
- [ ] Kindly add a `link` attribute to every project object and put inside it a link to the live demo.
- [ ] kindly change `languages` attribute to `technologies`.
- [ ] kindly write the correct path of the project's image so you can use it in the pop-up window.